src/pages/en.md

```diff
@@ -316,9 +316,16 @@ For example, Rails has a [`rails-dev-box`](https://github.com/rails/rails-dev-bo
 
 [Docker](https://www.docker.com) is also a good way to make dependency management simpler.
 
-- [ ] Promptly update version of gem at RubyGems
+- [ ] Don't forget to update version of gem at RubyGems
 
-You need to keep version of gem at [RubyGems](https://rubygems.org/) is up-to-date.
+You need to keep version of gem at [RubyGems](https://rubygems.org/) up-to-date. You should use this clarification when make decision about updating:
+
+- Security fix – release it!
+- Fixing a regression bug – release it! (Example: Ruby 2.3.3)
+- Adding a feature – wait for next planned release.
+- Fixing a bug that was there for a long time – wait for next planned release.
+
+***Note**: publish release candidate/beta version prior to releasing a major update.*
```

Comment:
"Its's a good practice to publish RC/beta versions prior to releasing a major update"